docs/source/conf.py

```diff
@@ -363,10 +367,10 @@
                  'figure.subplot.top': 0.96,
                  'figure.subplot.left': 0.04,
                  'figure.subplot.right': 0.96}
-plot_formats = (('thumb.png', 20),
+plot_formats = [('thumb.png', 20),
```

Comment:
This kept giving me a warning when building the site saying it should be a list, so I updated it